test/mdflatten.jl

```diff
@@ -0,0 +1,77 @@
+module MDFlattenTests
+
+if VERSION >= v"0.5.0-dev+7720"
+    using Base.Test
+else
+    using BaseTestNext
+    const Test = BaseTestNext
+end
+
+using Base.Markdown
+using Documenter.Utilities.MDFlatten
+
+@testset "MDFlatten" begin
+    @test mdflatten(Markdown.Paragraph("...")) == "..."
+    @test mdflatten(Markdown.Header{1}("...")) == "..."
+
+    # a simple test for blocks in top-level (each gets two newline appended to it)
+    @test mdflatten(Markdown.parse("# Test\nTest")) == "Test\n\nTest\n\n"
+    block_md = md"""
```

Comment:
Can `@md_str` be replaced with `Markdown.parse` in this and the cases below? I'd prefer only using `Markdown.parse` if possible.